app/assets/javascripts/controllers/dialog_editor/dialog_editor_controller.js

```diff
@@ -7,8 +7,11 @@ ManageIQ.angular.app.controller('dialogEditorController', ['$window', 'API', 'mi
         'dialog_tabs': [{
           'label': 'New tab',
           'position': 0,
-          'dialog_groups': [
-          ],
+          'dialog_groups': [{
+            'label': 'New box',
+            'position': 0,
+            'dialog_fields': []
```

Comment:
@romanblanco could get rid of the warning if you add a comma ... otherwise, maybe we should turn off this check. :)